.editorconfig

```diff
@@ -0,0 +1,17 @@
+# This file is for unifying the coding style for different editors and IDEs.
+# More information at http://EditorConfig.org
+
+# No .editorconfig files above the root directory
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 120
+
+[*.{js,json}]
+max_line_length = off
```

Comment:
That neat little red marker means: no blank new line at the end of file aka **wrong**